app.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import aws_cdk as cdk
+import aws_data_landing_zone as dlz
+from data_landing_zone_example_python.config_minimum import config
+
+app = cdk.App()
+
+dlz.DataLandingZone(
```

Comment:
Can you not just pass the config file like in the TS example? https://github.com/DataChefHQ/aws-data-landing-zone-example-typescript/blob/main/bin/data-landing-zone-example-typescript.ts#L8